src/crypto/index.js

```diff
@@ -284,9 +284,17 @@ export const validateMnemonic = bip39.validateMnemonic
  */
 export const getPrivateKeyFromMnemonic = (mnemonic, derive = true, index = 0, password = '') => {
 
-  if (!bip39.validateMnemonic(mnemonic)) {
-    throw new Error("wrong mnemonic format")
+  if (!bip39.validateMnemonic(mnemonic, bip39.wordlists.egnlish)
```

Comment:
`egnlish`?

Reply:
Sorry for the mistake. And actually is funny. bip39.wordlists.egnlish is undefined, and the func will use the default which is bip39.wordlists.english. So it still works.  I will close this pull and correct it and do it again.